pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -387,13 +399,14 @@ in buildPythonPackage {
     tensorflow-tensorboard_2
   ];
 
-  nativeBuildInputs = lib.optional cudaSupport addOpenGLRunpath;
-
   postFixup = lib.optionalString cudaSupport ''
+    # NOTE: Switch back to `addOpenGLRunpath "$lib"` and the built-in `patchelf`
+    # once https://github.com/NixOS/patchelf/pull/256 is merged and released as
+    # the default version in nixpkgs.
+    patchelf=${patchelf}/bin/patchelf
```

Comment:
Please use nativeBuildInputs.

Reply:
Will using nativeBuildInputs override the original patchelf in PATH the way that I need it to?

Reply:
Tested and it works. Done in https://github.com/NixOS/nixpkgs/pull/137069/commits/fefedd02e6458ded537cfafa53f1fcdd5e5be076.